style.css

```diff
@@ -0,0 +1,102 @@
+.nav__link {
+	cursor: pointer;
+	color: black;
+	padding: 15px 10px;
+	text-decoration: none;
+	text-align: center;
+	font-size: 36px;
+}
+
+.burger {
+  position: relative;
+  display: none;
+  left: 0;
+  top: 5px;
+  content: '';
+  width: 25px;
+  height: 25px;
+  border-radius: 50%;
+  background: black;
+  opacity: 50%;
+}
+
+.burger::before {
+	content: '';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Azez1/pullRequest?prid=5948917)